ghost/email-service/lib/BatchSendingService.js

```diff
@@ -360,14 +360,19 @@ class BatchSendingService {
 
         // Loop batches and send them via the EmailProvider
         let succeededCount = 0;
-        const queue = batches.slice();
+        const queue = batches.slice().map((batch, index) => {
+            return {batchIndex: index, ...batch};
+        });
 
         // Bind this
         let runNext;
+        let startTime = new Date();
         runNext = async () => {
             const batch = queue.shift();
             if (batch) {
-                if (await this.sendBatch({email, batch, post, newsletter, emailBodyCache})) {
+                const deliveryDelay = batch.batchIndex * 5000; // 5 seconds delay between each batch
```

Comment:
Reminder to self: extract the hardcoded 5000 into a configuration parameter

Reply:
Is 5s a suitable value? If we have 100 batches, they could be spread up to 500s (8 mins) apart. Want to make sure this won't cause more problems due to a perceived delay in deliverability 🙂